packages/manager/modules/pci/src/projects/project/additional-ips/additional-ips.routing.js

```diff
@@ -25,53 +23,105 @@ export default /* @ngInject */ ($stateProvider) => {
         $q,
         OvhApiCloudProject,
         OvhApiCloudProjectIpFailover,
         projectId,
+        getFloatingIps,
       ) =>
         $q
           .all({
-            additionalIps: OvhApiCloudProjectIpFailover.v6().query({
+            failoverIps: OvhApiCloudProjectIpFailover.v6().query({
               serviceName: projectId,
             }).$promise,
             instances: OvhApiCloudProject.Instance()
               .v6()
               .query({
                 serviceName: projectId,
-              }).$promise,
+              })
+              .$promise.then((instances) =>
+                instances.reduce((accumulator, instance) => {
+                  accumulator[instance.id] = instance;
+                  return accumulator;
+                }, {}),
+              ),
+            floatingIps: getFloatingIps(),
           })
-          .then(({ additionalIps, instances }) =>
-            map(additionalIps, (additionalIp) => ({
-              ...additionalIp,
-              instance: additionalIp.routedTo
-                ? find(instances, { id: additionalIp.routedTo })
-                : null,
-            })),
-          ),
+          .then(({ failoverIps, instances, floatingIps }) => {
+            return {
+              failoverIps: map(failoverIps, (failoverIp) => ({
+                ...failoverIp,
+                instance: failoverIp.routedTo
+                  ? instances[failoverIp.routedTo]
+                  : null,
+              })),
+              floatingIps: map(floatingIps, (floatingIp) => ({
+                ...floatingIp,
+                instance:
+                  floatingIp.associatedEntity?.type === 'instance'
+                    ? instances[floatingIp.associatedEntity?.id]
+                    : null,
+              })),
+            };
+          }),
+      currentActiveLink: /* @ngInject */ ($transition$, $state) => () =>
+        $state.href($state.current.name, $transition$.params()),
+
+      failoverIpsLink: /* @ngInject */ ($state, projectId) =>
+        $state.href('pci.projects.project.additional-ips.failover-ips', {
+          projectId,
+        }),
 
-      additionalIpsRegions: /* @ngInject */ (additionalIps) =>
-        Array.from(new Set(additionalIps.map(({ geoloc }) => geoloc))),
+      floatingIpsLink: /* @ngInject */ ($state, projectId) =>
+        $state.href('pci.projects.project.additional-ips.floating-ips', {
+          projectId,
+        }),
+
+      getFloatingIps: /* @ngInject */ (
+        $http,
+        $q,
+        OvhApiCloudProjectRegion,
+        projectId,
+      ) => () =>
+        OvhApiCloudProjectRegion.v6()
+          .query({
+            serviceName: projectId,
+          })
+          .$promise.then((regions) => {
+            return $q
+              .all(
+                regions.map((region) =>
+                  $http
+                    .get(
+                      `/cloud/project/${projectId}/region/${region}/floatingip `,
```

Comment:
**thought:** API are still flagged as internal.

Reply:
The prod of `l3-services` is planned for September. Hoping by that time the APIs will available.